node.gni

```diff
@@ -62,6 +62,12 @@ declare_args() {
 
   # Build with Amaro (TypeScript utils).
   node_use_amaro = true
+
+  # Allows downstream packagers (eg. Linux distributions) to build against system shared libraries.
+  use_system_cares = false
```

Comment:
What about Ada?

Reply:
I'd say it's a small user group because Ada uses C++ types that are not ABI-compatible between LLVM libc++ (linked statically by Chromium as default) and libstdc++ (used as the system one by most distros). `use_custom_libcxx=false` is very broken, and pretty much the only distros I know of using it currently are Chimera Linux (which uses libc++ as system, and patches _libc++_ to keep Chromium working), Debian, and openSUSE

Reply:
That's educational. Thanks!

Reply:
To clarify: openSUSE uses `use_custom_libcxx=false` for Electron (which i maintain), but not for Chromium (which i don't, and don't have resources to).

It's much more important that it stays usable for Electron, since requiring a custom C++ implementation would also require changes in packaging every Electron application we ship that has native modules.